src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    protected static final String type = "rectangle";
+    protected double width;
+    protected double height;
```

Comment:
The 'width' and 'height' fields should be private to adhere to encapsulation principles. Also, consider if they need to be protected, as this allows subclasses to access them directly, which might not be desired.